src/DAOApi.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import type { Headers } from 'brewskey.js-api';
+
+import oHandler from 'odata';
+
+import AccountDAO from './dao/AccountDAO';
+import AvailabilityDAO from './dao/AvailabilityDAO';
+import BeverageDAO from './dao/BeverageDAO';
+import DeviceDAO from './dao/DeviceDAO';
+import GlassDAO from './dao/GlassDAO';
+import KegDAO from './dao/KegDAO';
+import LocationDAO from './dao/LocationDAO';
+import PermissionDAO from './dao/PermissionDAO';
+import PourDAO from './dao/PourDAO';
+import ScheduleDAO from './dao/ScheduleDAO';
+import SrmDAO from './dao/SrmDAO';
+import StyleDAO from './dao/StyleDAO';
+import TapDAO from './dao/TapDAO';
+
+type OConfig = {
+  endpoint: string,
+  headers?: Headers,
+};
+
+class DAOApi {
+  AccountDAO: typeof AccountDAO;
```

Comment:
this only for flow.
I can import types instead use `typeof`..but this will be just 15 lines imports for nothing